packages/aws-rfdk/lib/deadline/lib/render-queue.ts

```diff
@@ -214,15 +215,21 @@ export class RenderQueue extends RenderQueueBase implements IGrantable {
   */
   private static readonly MINIMUM_LOAD_BALANCING_VERSION = new Version([10, 1, 10, 0]);
 
+  /**
+   * The minimum Deadline version required to enable Deadline Secrets Management on the Render Queue.
+   */
+  private static readonly MINIMUM_SECRETS_MANAGEMENT_VERSION = new Version([10, 1, 18, 0]);
```

Comment:
This is tricky to add at this point because there are features being added to Deadline to make Secrets Management easier to use that are still unreleased, and we don't yet support `10.1.18` in RFDK, so adding this here will render our feature branch unusable. 

Maybe let's set this to 10.1.15 for now (that release had a fix for when RCS is behind a load balancer that is absolutely required) and we can update it after Deadline releases the new features and before we merge this from the feature branch to mainline.

Reply:
Sounds good, I've put a TODO in there for now so we don't forget to do that before we merge this into mainline